core/services/run_manager.go

```diff
@@ -389,13 +396,16 @@ func (jm *runManager) updateWithError(run *models.JobRun, msg string, args ...in
 		logger.Errorw("Error saving run", run.ForLogger("error", err)...)
 		return err
 	}
+	jm.statsPusher.PushNow()
 	return nil
 }
 
 func (jm *runManager) updateAndTrigger(run *models.JobRun) error {
+	defer jm.statsPusher.PushNow()
```

Comment:
Not related to this PR but, what's the difference between a `JobManager` (jm) and a `RunManager` (the type itself)?

Reply:
There is no JobManager, this is just a relic.